app/assets/stylesheets/petitions/_search-inline.scss

```diff
@@ -30,5 +30,8 @@
     background-repeat: no-repeat;
     background-position: 45% 50%;
     @include background-image("icons/icon_search-white", 18px, 18px);
+    &:hover, &:focus {
```

Comment:
Would prefer not to use the & scss pattern. It visually implies a nested element.

Better to just use a flat `.thing:hover`